### PR TITLE
Add song Regalskeppet Vasa

### PR DESCRIPTION
**Pull request by Cactooz**

### CODE REVIEW

File: songs/153_Regalskeppet_Vasa.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Regalskeppet Vasa
+author: Den Konliga INviten (Ø)
```
Comment (StelFoog) [inline songs/153_Regalskeppet_Vasa.md:3]:
```suggestion
author: Den Kongliga INviten
```

**Review by StelFoog — CHANGES_REQUESTED** [review]
Looks good, just one small fix I'd like

File: songs/153_Regalskeppet_Vasa.md (hunk 1 above)
Comment (Cactooz) [reply songs/153_Regalskeppet_Vasa.md]: Problem solved!

**Review by StelFoog — APPROVED** [review]
LGTM

**Pull request merged**